### PR TITLE
set cpu request for operator

### PR DESCRIPTION
**Pull request by sjenning**
@smarterclayton @derekwaynecarr

### CODE REVIEW

**Comment by smarterclayton** [comment]
I'm ok with 20 because this is the root of the tree

/lgtm

File: install/0000_00_cluster-version-operator_03_deployment.yaml [hunk 1]
```diff
@@ -22,6 +22,10 @@ spec:
           - "--release-image={{.ReleaseImage}}"
           - "--enable-auto-update=false"
           - "--v=4"
+        resources:
+          requests:
+            cpu: 20m
```
Comment (derekwaynecarr) [inline install/0000_00_cluster-version-operator_03_deployment.yaml]: Why 20?

**Pull request merged**